runtime/integration-tests/Cargo.toml

```diff
@@ -99,6 +100,11 @@ xcm-executor = { git = "https://github.com/paritytech/polkadot", default-feature
 
 pallet-collator-selection = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.38" }
 
+ethereum = { version = "0.14.0", default-features = false }
+
+pallet-evm = { git = "https://github.com/PureStake/frontier", default-features = false, branch = "moonbeam-polkadot-v0.9.38" }
+pallet-evm-chain-id = { git = "https://github.com/PureStake/frontier", default-features = false, branch = "moonbeam-polkadot-v0.9.38" }
```

Comment:
For dev dependencies, we want `std`. It's enough just remove the `default-features = false` part.

Not sure if we need to add `runtime-benchmarks` for these pallets too.

Could you test if:
```sh
cargo test -p `runtime-integration-tests` -F runtime-benchmarks
```
works?

Reply:
I'll remove the `default-features` and try it out.

Reply:
`cargo test -p `runtime-integration-tests` -F runtime-benchmarks` worked successfully.